code/game/turfs/closed/minerals.dm

```diff
@@ -80,6 +82,23 @@
 	else
 		return attack_hand(user)
 
+/turf/closed/mineral/attack_hand(mob/user)
+	if(weak_turf)
+		var/turf/T = user.loc
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names